### ISSUE TITLE
Invisible Sliders and Dropdown Box in Settings

### ISSUE THREAD

**Issue by zaidnaseer** [RhythmPahwa14/AlgoVisualizer#1215]
### Describe the bug

Sliders in the Settings -> Accessibility section are not visible clearly when they are in OFF state. 

<img width="511" height="721" alt="Image" src="https://github.com/user-attachments/assets/212e6f57-c010-421d-81cd-e70eeb4c4e10" />

Also, dropdown boxes in the Settings -> Typography and Language section are not visible
<img width="517" height="672" alt="Image" src="https://github.com/user-attachments/assets/0927a51e-41fd-4fe1-b7b2-6bb3a394cfc6" />
